js/chatbot/shared/Component.svelte

```diff
@@ -27,6 +34,27 @@
 		fixed_height={1}
 		on:load
 	/>
+{:else if type === "dataframe"}
+	<svelte:component
+		this={components[type]}
+		{value}
+		show_label={false}
+		{i18n}
+		label=""
+		{_fetch}
+		allow_preview={false}
+		interactive={false}
+		mode="minimal"
+		fixed_height={1}
+		line_breaks={true}
```

Comment:
I'm curious why why we force certain parameters like `line_break={true}` instead of doing:
```suggestion
		line_breaks={props.line_breaks}
```